plone/schemaeditor/__init__.py

```diff
@@ -1,2 +1,4 @@
 from zope.i18nmessageid import MessageFactory
-SchemaEditorMessageFactory = MessageFactory('plone.schemaeditor')
```

Comment:
Removing this definition breaks packages with import it. E.g. it breaks compatibility with plone.app.dexterity < 2.1:
https://github.com/plone/plone.app.dexterity/blob/2.0.x/plone/app/dexterity/browser/modeleditor.py#L11

This version of schemaeditor and plone.app.dexterity are only used in combination if you use specific version fixtures.

But your change should have made it into a new minor release, not a bugfix release.